File: Modules/NodeJS/Day10/Assignment/Assignment/mongoDb/empCollection/employees.js
const mongoose = require('mongoose')

mongoose.connect('mongodb://localhost/Students',{useNewUrlParser: true, useUnifiedTopology: true})
    .then()
    .catch(err => console.log('could not connect to server .....',err))

const employeeSchema = mongoose.Schema({
    AddressLine1: String,
    AddressLine2: String,
    AddressLine3: String,
    assignments: [ Number ],
    CitizenshipId: Number ,
    CitizenshipLegislationCode: String,
    CitizenshipStatus: String,
    CitizenshipToDate: Date,
    City: String,
    CorrespondenceLanguage: String,
    Country: String,
    CreationDate: Date,
    DateOfBirth: Date,
    directReports: [ Number ],
    DisplayName: String,
    DriversLicenseExpirationDate: Date,
    DriversLicenseId: Number ,
    DriversLicenseIssuingCountry: String,
    EffectiveStartDate: Date,
    Ethnicity: String,
    FirstName: String,
    Gender: String,
    HireDate: Date,
    HomeFaxAreaCode: String,
    HomeFaxCountryCode: String,
    HomeFaxExtension: String,
    HomeFaxLegislationCode: String,
    HomeFaxNumber: String,
    HomePhoneAreaCode: String,
    HomePhoneCountryCode: String,
    HomePhoneExtension: String,
    HomePhoneLegislationCode: String,
    HomePhoneNumber: String,
    Honors: String,
    LastName: String,
    LastUpdateDate: Date,
    LegalEntityId: Number ,
    LicenseNumber: String,
    links: [ Number ],
    MaritalStatus: String,
    MiddleName: String,
    MilitaryVetStatus: String,
    NameSuffix: String,
    NationalId: String,
    NationalIdCountry: String
})

const Employees = mongoose.model('Employees',employeeSchema)

module.exports = Employees

File: Modules/C#/Day15/Charmi Tanna/Assignment/Assignment1/Assignment1/Models/Employees.cs

using Assignment1.Models;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Assignment1
{
    public class Employees
    {
        [Key]
        public int EmployeeId { get; set; }
        public string FirstName { get; set; }
        public string MiddleName { get; set; }
        public string LastName { get; set; }
        public string Gender { get; set; }
        public string AddressLine1 { get; set; }
        public string AddressLine2 { get; set; }
        public string AddressLine3 { get; set; }
        public string City { get; set; }
        public string HireDate { get; set; }
        public virtual ICollection<Assignments> Assignments { get; set; }

    }
}
